packages/pages-components/src/map/coordinate.ts

```diff
@@ -144,15 +120,19 @@ function mercatorLatAddRadians(startingLat, radians) {
  * Longitude is a degree number without limits but is normalized to [-180, 180).
  */
 class Coordinate {
+  _lat: number;
+  _lon: number;
   /**
    * Constructor takes either 1 or 2 arguments.
    * 2 arguments: latitude and longitude.
-   * 1 argument: an object with at least one {@link module:@yext/components-geo~Coordinate~LATITUDE_ALIASES latitude alias}
-   * and one one {@link module:@yext/components-geo~CoordinateCoordinate~LONGITUDE_ALIASES longitude alias}.
-   * @param {number|Object} latitudeOrObject
-   * @param {number} [longitude] Optional only if the first argument is a {@link module:@yext/components-geo~Coordinate Coordinate}-like object
+   * 1 argument: an object with at least one {@link LATITUDE_ALIASES | latitude alias}
+   * and one one {@link LONGITUDE_ALIASES | longitude alias}.
+   * @param longitude - Optional only if the first argument is a {@link Coordinate}-like object
    */
-  constructor(latitudeOrObject, longitude) {
+  constructor(
+    latitudeOrObject: number | { [key: string]: any },
```

Comment:
Why is latitude a different type than longitude? Should they both be a number or object? Weird that this is called latitudeOrObject and then the or is a string->any, but then longitude isn't called longitudeOrObject and then the or is some other named type which is a function that returns a number.

Reply:
That was the existing setup/ I didn't change it so I'm not sure why. But I think the idea is that someone would either pass two numeric params for lat and lon, or a single param that's an object in a format like { lat: x, lon: x } . Happy to make any additional changes to it that you recommend.

Reply:
I agree, it does seem weird. I did a quick search for the repo and don't see anywhere that passes a function as a param to this class so I'm not sure what the use case would be for it. Do you think I should remove it?

Reply:
I think it should be updated as long as none of the map providers return a function like that. That would be my only guess as to why it was written that way.